keps/prod-readiness/sig-storage/2923.yaml

```diff
@@ -1,3 +1,3 @@
 kep-number: 2923
-alpha:
-  approver: "@ehashman"
\ No newline at end of file
+beta:
```

Comment:
Instead of replacing alpha with beta, it should be a new entry for beta.

Reply:
@msau42 sure, added it as a new entry.. ptal.. thanks.

Reply:
Done.. thanks.